src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -634,13 +635,15 @@ bool BaseCouplingScheme::doImplicitStep()
   // coupling iteration converged for current time window. Advance in time.
   if (convergence) {
     if (_acceleration) {
-      _acceleration->iterationsConverged(getAccelerationData());
+      auto accelerationData = getAccelerationData();
+      _acceleration->iterationsConverged(accelerationData);
     }
     newConvergenceMeasurements();
   } else {
     // no convergence achieved for the coupling iteration within the current time window
     if (_acceleration) {
-      _acceleration->performAcceleration(getAccelerationData());
+      auto accelerationData = getAccelerationData();
+      _acceleration->performAcceleration(accelerationData);
```

Comment:
This shouldn't be necessary if you adjust the interface of Acceleration. #1194

Reply:
Exactly :+1: 039a23b6926fc7bbca46348b33ca73a59f3ba3c8